rts/Lua/LuaVFS.cpp

```diff
@@ -38,6 +38,9 @@
  * Fortunately, the Spring Lua system automatically provides access to mod and
  * base files via the VFS module.
  *
+ * As an additional caveat, using VFS is mandatory for synced, which cannot
```

Comment:
This sentence sounds puzzling to me, what's mandatory for synced, using VFS only within a particular set of modes?

Reply:
How about now?

Reply:
Looks better, ty